### ISSUE TITLE
Implement general services

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#51]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

* File service
* User model service
* Message type enum
* User message view model
* Page view model
* User message display template, view

### Why

To support cross-cutting concerns in the Identity codebase.

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
